docs/book/v2/getting-started/skeleton.md

```diff
@@ -615,13 +616,17 @@ For the next part of the example, we'll be creating and wiring a factory for
 creating the `HelloAction` instance; the example assumes you used the default
 selection for a dependency injection container, zend-servicemanager.
 
-zend-servicemanager provides a tool for generating factories based on
-reflecting a class; we'll use that to generate our factory:
+zend-servicemanager provides a console (command line) tool for generating factories 
```

Comment:
The target audience of this documentation are developers and therefore "console tool" is enough.
(The link to the `generate-factory-for-class` command should be added here.)